docs/src/main/sphinx/connector/thrift.rst

```diff
@@ -14,6 +16,13 @@ splits and data. The connector randomly chooses a server to talk to from the ava
 instances for metadata calls, or for data calls unless the splits include a list of addresses.
 All requests are assumed to be idempotent and can be retried freely among any server.
 
+Requirements
+------------
+
+To connect to your custom servers with the Thrift protocol, you need:
+
+* Network access from the Trino coordinator and workers to the Thrift servers.
```

Comment:
Also required is the step of implementing your own Thrift interface.

See https://github.com/trinodb/trino/pull/8377/files#diff-33aa44cbc9e5070dad299e4d6c9815bfc5b3eac7800ce6efa4f8453955a6c813R96-R102.

> Thrift IDL file
> ---------------
> The following IDL describes the ``TrinoThriftService`` that must be implemented:
> .. literalinclude:: /include/TrinoThriftService.thrift
>     :language: thrift

Reply:
May be worded as ```An implementation of ``TrinoThriftService`` which is described by the Thrift IDL at `/include/TrinoThriftService.thrift` ```